roles/core/set_hostname/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+set_hostname_fully_qualified_domaine_name: false
```

Comment:
Could you define a shorter parameter name? `set_hostname_fqdn`?

Reply:
Done 🙂